dbt/include/dremio/macros/adapters/persist_docs.sql

```diff
@@ -0,0 +1,20 @@
+-- {% macro persist_docs(relation, model, for_relation=true, for_columns=true) -%}
```

Comment:
Do we need all these rows commented? Could these lines be deleted from this file?